code/game/objects/items/weapons/candle.dm

```diff
@@ -12,11 +12,11 @@
 
 /obj/item/flame/candle/Initialize(mapload)
 	. = ..()
-	wax -= rand(800, 1000) // Enough for 27-33 minutes. 30 minutes on average.
+	wax -= 7200 //CHOMPEdit, FOUR HOUR burn time, taking into account process only calling once every two seconds or so. Originally was: rand(800, 1000) // Enough for 27-33 minutes. 30 minutes on average.
```

Comment:
We're removing the wax here, this will make the wax go from 2000 to -5200 and immediately qdel